frontend/src/styles/notif_bar.scss

```diff
@@ -0,0 +1,4 @@
+.nav-tabs {
+    display: inline-flex !important;
```

Comment:
I would avoid using !important unless you're overriding inline styling. Otherwise it's always better to override CSS by using specificity